File: src/Actors/Usuario.java
package Actors;

public class Usuario {

    private String nombre;
    private String apellidos;
    private String telefono;
    private String identificacion;
    private String correo;
    private String contrasenia;
    private String rol;

    public Usuario(String nombre, String apellidos, String telefono, String identificacion, String correo, String contrasenia, String rol) {
        this.nombre = nombre;
        this.apellidos = apellidos;
        this.telefono = telefono;
        this.identificacion = identificacion;
        this.correo = correo;
        this.contrasenia = contrasenia;
        this.rol = rol;
    }

    public String getNombre() {
        return nombre;
    }

    public String getApellidos() {
        return apellidos;
    }

    public String getTelefono() {
        return telefono;
    }

    public String getIdentificacion() {
        return identificacion;
    }

    public String getCorreo() {
        return correo;
    }

    public String getContrasenia() {
        return contrasenia;
    }

    public String getRol() {
        return rol;
    }

    @Override
    public String toString() {
        return "Usuario{" + "nombre=" + nombre + ", apellidos=" + apellidos + ", telefono=" + telefono + ", identificacion=" + identificacion + ", correo=" + correo + ", contrasenia=" + contrasenia + ", rol=" + rol + '}';
    }
    
}

File: Códigos/Actores/Usuario.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ProyectoEstructuras.Códigos.Actores
{
    public class Usuario
    {
        private string nombre;
        private string apellidos;
        private string telefono;
        private string identificacion;
        private string correo;
        private string contrasenia;
        private string rol;

        public string Nombre { get => nombre; set => nombre = value; }
        public string Apellidos { get => apellidos; set => apellidos = value; }
        public string Telefono { get => telefono; set => telefono = value; }
        public string Identificacion { get => identificacion; set => identificacion = value; }
        public string Correo { get => correo; set => correo = value; }
        public string Contrasenia { get => contrasenia; set => contrasenia = value; }
        public string Rol { get => rol; set => rol = value; }

        public Usuario(string nombre, string apellidos, string telefono, string identificacion, string correo, string contrasenia, string rol)
        {
            this.Nombre = nombre;
            this.Apellidos = apellidos;
            this.Telefono = telefono;
            this.Identificacion = identificacion;
            this.Correo = correo;
            this.Contrasenia = contrasenia;
            this.Rol = rol;
        }

        public override string ToString()
        {
            return $"Usuario{{nombre={Nombre}, apellidos={Apellidos}, telefono={Telefono}, identificacion={Identificacion}, correo={Correo}, contrasenia={Contrasenia}, rol={Rol}}}";
        }
    }
}
